src/slashcommands/BirthdayList.ts

```diff
@@ -0,0 +1,156 @@
+import { CommandInteraction, CacheType, MessageEmbed, Message, User, MessageReaction } from 'discord.js';
+import { Bot } from '../Bot';
+import { Option, Subcommand } from './Option';
+import { SlashCommand} from './SlashCommand';
+import { bdayDates } from './Birthday';
+
+type monthIndex = { [index: number]: string};
+const monthCode = {
+	1: 'January',
+	2: 'February',
+	3: 'March',
+	4: 'April',
+	5: 'May',
+	6: 'June',
+	7: 'July',
+	8: 'August',
+	9: 'September',
+	10: 'October',
+	11: 'November',
+	12: 'December',
+} as monthIndex;
+
+export class BirthdayList implements SlashCommand {
+    name: string = 'birthdaylist'
+    description: string = '[MANAGER] See all the birthdays in the current guild'
+    options: (Option | Subcommand)[] = [];
+    requiredPermissions: bigint[] = [];
+    async run(bot: Bot, interaction: CommandInteraction<CacheType>): Promise<void> {
+        await interaction.deferReply();
+        bdayDates.fetchEverything();
+        let list: any[][] = [];
+        for(const date of bdayDates){
+            try{
```

Comment:
This loop is so slow, try implementing https://stackoverflow.com/questions/53278791/check-if-user-id-exists-in-discord-server